HospitalSystem.ConsoleUI/Doctor/DoctorCheckupsUI.cs

```diff
@@ -7,8 +7,12 @@ namespace HospitalSystem.ConsoleUI;
 
 public class DoctorCheckupsUI : DoctorMainUI
 {
```

Comment:
Maybe put this in the Doctor namespace so you don't gave to prefix every class name with Doctor?

Reply:
Okay